molgenis/molgenis
# M4766 Virtual page views in the dataexplorer

## How to test
- Go the the [localhost GA page](https://analytics.google.com/analytics/web/#management/Settings/a60045777w163212861p164151179/%3Fm.page%3DTrackingCode/)
- Add the tracking code to the MOLGENIS tracking ID field
- Go to the dataexplorer
- use filters
- see that page views in GA realtime is updated

#### Checklist
